### PR TITLE
Revert "New KW `Wait For Deployment Replica To Be Ready` for oc_install.robot" because discussion was not done 

### PR DESCRIPTION
**Pull request by tarukumar**
Reverts red-hat-data-services/ods-ci#1867

### CODE REVIEW

**Comment by tarukumar** [comment]
why you guys will merge the pr when discussion was still on

**Comment by tarukumar** [comment]
what if i go ahead and get two approval and merge the pr. Kindly follow the process and do not decide on the fly until all the discussion is done

**Review by jstourac — APPROVED** [review]
Approving this in case we decided we want to revert this before the true and proper tests are implemented elsewhere.

It would be also nice to enrich this with the extra commit amending the number of expected pods for the `odh-model-controller` from 3 to 1 on those two places where it is checked here. But can be also as a separate PR.

**Pull request closed without merging**
